Different log format on console and in file ( v/vlib/log/log.v )

**V version:** 
V 0.2.2 205fb88

**OS:** 
OS: linux, Ubuntu 20.04.2 LTS

**What did you do?**
Check logging into console. Check logging into file. 

**What did you expect to see?**
The output in the same format.
 
**What did you see instead?**
Console (with colored INFO, WARN, etc.):
[INFO  2021-04-12 04:07:49] info
[WARN  2021-04-12 04:07:49] warn
[ERROR 2021-04-12 04:07:49] error
File:
2021-04-12 03:11:31 [INFO ] info
2021-04-12 03:11:31 [WARN ] warn
2021-04-12 03:11:31 [ERROR] error

The **same** format (as in file, colors as an option) is preferable in both cases.

--- a/vlib/log/log.v
+++ b/vlib/log/log.v
@@ -126,9 +126,9 @@ fn (mut l Log) log_file(s string, level Level) {
 
 // log_cli writes log line `s` with `level` to stdout.
 fn (l &Log) log_cli(s string, level Level) {
-	f := tag_to_cli(level)
-	t := time.now()
-	println('[$f $t.format_ss()] $s')
+	timestamp := time.now().format_ss()
+	e := tag_to_cli(level)
+	println('$timestamp [$e] $s')
 }
 
 // send_output writes log line `s` with `level` to either the log file or the console

